### PR TITLE
Natural language date parser added.

### PR DESCRIPTION
**Pull request by RajPratim21**
@oliversauter  @Treora , I have added the Natural language date parser , using moment.js
naturalparser

### CODE REVIEW

File: src/search/find-visits.js [hunk 1]
```diff
@@ -75,42 +77,40 @@ export function findVisitsToPages({pagesResult}) {
      * Raj Pratim Bhattacharya gmail rajpratim1234@gmail.com
      */
     
-    var comp = new Date();
-    var sDate = comp.getTime() - 100*24*60*60*1000 //100 days old search
-    var eDate =  comp.getTime()
-    
+
+    var current_time = new moment().valueOf();
+    var startDate = current_time - 100*24*60*60*1000; //100 days old search
```
Comment (obsidianart) [inline src/search/find-visits.js]: why not using moment -days like in the other?

Comment (RajPratim21) [reply src/search/find-visits.js]: @obsidianart  I am afraid , I coudn't get you , can you refer where I have used `moment-days` previously.

Comment (obsidianart) [reply src/search/find-visits.js]: minDate={moment().subtract(365,"days")}

Comment (RajPratim21) [reply src/search/find-visits.js]: @obsidianart , I will be adding a commit based on the recent commit by @Treora  in #76 for date-range picker to maintain the style. and easier evaluation.

Comment (obsidianart) [reply src/search/find-visits.js]: I'm not sure I understand your comment.

Comment (RajPratim21) [reply src/search/find-visits.js]: I meant to say , this commit is made (i.e natural date parser is added ) in the same way I added the date range pickers earlier , as recently @Treora , added some changes to the date-range picker ,in #76 , I will make same changes here to make it consistent wiith the style of implementation.

**Comment by Treora** [comment]
This has not been worked on in a while, would still need quite some review and further work (e.g. integrating in the query string rather than having a separate input box), and is one of those fancy features that I would rather not dilute my [focus](https://blog.webmemex.org/2017/04/24/progress-update/#%22Increasing%20focus%22) with. If anybody wants to pick it up and finish it all the way, of course feel free to do so and send a new PR, but I hope you don't mind me closing this one.

**Comment by RajPratim21** [comment]
@Treora  I have been a bit busy in last few weeks due to my exams I will try to reimplement it and send a new PR with the things mentioned, but if it seemed too much work for a fancy not so importaant feature, I may skip it to focus more on working with search performance.

**Pull request closed without merging**